packages/liveblocks-yjs/src/doc.ts

```diff
@@ -105,6 +135,8 @@ export default class yDocHandler extends Observable<unknown> {
     update: Uint8Array,
     origin: string | IndexeddbPersistence
   ) => {
+    this.localSnapshotΣ.set(Y.snapshot(this.doc));
```

Comment:
again, I'm worried about the perf, especially with large documents. Doing this on every single update seems rather brute force.